src/components/button/Button.scss

```diff
@@ -0,0 +1,37 @@
+@use "../../../src/styles/reset" as *;
+@use "../../../src/styles/variables" as *;
+@use "../../styles/typography" as *;
+@use "../../styles/mixins" as *;
+
+.button {
+  background-color: $background-color;
+
+  width: 9rem;
+
+  height: 3rem;
+
+  font-size: $default-text-size;
+
+  padding: 10px 20px;
+
+  border: solid black;
+}
+
+.button-contact {
+  margin: auto;
+  width: 12rem;
+  font-family: $text-Font-Family;
+  color: $white;
+  margin-top: 10px;
+  padding: 0.5rem 0.9rem;
+
+  background-color: $green-color-palette;
+
+  font-size: 3rem;
+
+  font-weight: $default-font-weight;
+
+  border-radius: 25px;
+
+  border: none;
```

Comment:
remove whitespace between these lines.. you can run `npm run format` to automatically do your formatting